complete_article_data.csv

```diff
@@ -0,0 +1,110 @@
+,Title,Author,Gender of Author,Sources,# of Male Sources,# of Female Sources
```

Comment:
We should avoid commiting compiled files -- let's make a data repository whose contents are `.gitignored`

So, add `data/*.csv` to `.gitignore`
And ad a `data/.gitkeep` which will allow git to actually store the folder